README.md

```diff
@@ -1,2 +1,2444 @@
-# web-ui
-User interface project
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
I would either remove this file or rename it to sort of README_react-create-app.md.
We should provide our own README file with related content.

But this can be done in a follow-up.

Reply:
:+1: to remove.

Reply:
OK, it will be renamed.